src/etl/test_mock_save_data.py

```
@@ -0,0 +1,43 @@
+
+        print(f"SAVED: {len(messages)} messages")
+
+        for msg in messages[:3]:
+            print(msg)
+
+import asyncio
+
+async def main():
+
+    # HTML parser
+    html_parser = HTMLParser()
```

Ruff (0.15.12)

[error] Undefined name `HTMLParser`

(F821)